Assets/RTLTMPro/Scripts/Runtime/TextUtils.cs

```diff
@@ -79,6 +79,12 @@ public static bool IsEnglishLetter(char ch)
                 || ch >= ArabicPresentationFormsBBlockLow && ch <= ArabicPresentationFormsBBlockHigh;
         }
 
+        // Checks if the character is a quote symbol. Either the generic " or guillemets
+        public static bool IsQuote(char input)
+        {
+            return input is '"' or '�' or '�';
```

Comment:
I would rather include those special chars as \uXXXX format rather than trust file encoding.

Reply:
It's less risky using numerical values, as you can accidentally break file encoding thus breaking those chars, but you cannot accidentally break numerical values.

Reply:
No, there's no diff, using numeric constants casted to char or directly using char literals makes no diff. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/char#literals